ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -182,6 +183,7 @@ export function WorkflowDetails({history, location, match}: RouteComponentProps<
     }, [nodeId, workflow]);
 
     const parsedSidePanel = parseSidePanelParam(sidePanel);
+    const {t} = useTranslation();
```

Comment:
same as above, should be with the `use` variable declarations at the top of the component